src/mlpack/tests/lsh_test.cpp

```diff
@@ -117,8 +117,10 @@ BOOST_AUTO_TEST_CASE(NumTablesTest)
   const string testSet = "iris_test.csv";
   arma::mat rdata;
   arma::mat qdata;
-  data::Load(trainSet, rdata, true);
-  data::Load(testSet, qdata, true);
+  if (!data::Load(trainSet, rdata, true)) 
```

Comment:
```suggestion
  if (!data::Load(trainSet, rdata)) 
```
Third argument not required.